### PR TITLE
Brandon porter

### PR DESCRIPTION
**Pull request by AshenPheonix**
Thursday's Project

### CODE REVIEW

**Comment by AshenPheonix** [comment]
I'm fairly sure the only thing Bootstrap is doing is controlling the navbar and the modal.

File: instagram-app/src/App.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+import {BrowserRouter as Router,Route, Link, Redirect} from 'react-router-dom'
 
 export default class extends React.Component {
   render(){
     return (
-      <ComponentWithAuthenticate />
+      <Router>
+        <PrivateRoute path='/' component={PostsPage}/>
+        <Route path='/login' component={Login}/>
+      </Router>
```
Comment (OmarSalah95) [inline instagram-app/src/App.js:13]: You actually need to use the `exact` flag to your route here. If you notice that both the login page and app are rendering once you get into the actual app (look at the bottom of the page).

File: instagram-app/src/components/PostContainer/PostCard.jsx [hunk 2]
```diff
@@ -1,35 +1,34 @@
 import React, { Component } from 'react'
 import User from './User'
 import CommentBox from '../CommentSection/Comments'
-import {FaRegHeart as Heart, FaRegComment as Comment} from 'react-icons/fa'
 import './Posts.scss'
+import {PostContainer, PostImage,PostClosure,LikeButton,CommentButton,LikeNumber} from '../../styled/PostCard'
```
Comment (OmarSalah95) [inline instagram-app/src/components/PostContainer/PostCard.jsx:5]: Great job importing all the styled components that are from the same file.

File: instagram-app/src/components/PostContainer/PostsPage.jsx [hunk 3]
```diff
@@ -49,6 +49,7 @@ export default class PostsPage extends Component {
       }
     
       render(){
+        console.log('loading page');
         return (
           <div className="App container">
```
Comment (OmarSalah95) [inline instagram-app/src/components/PostContainer/PostsPage.jsx:54]: I would have liked to have seen the username stored somewhere in state on login, this way you can dynamically add the logged in users username as the preface to their own comments on add.

File: instagram-app/src/components/SearchBar/SearchBar.jsx [hunk 4]
```diff
@@ -1,44 +1,47 @@
...
-                    <section className="collapse navbar-collapse d-md-flex justify-content-md-between" id="navbarSupportedContent">
-                        <section className="search">
-                            <FaSearch/>
-                            <input type="text" placeholder="Search" onChange={this.props.searchEdit} value={this.props.searchValue}/>
-                        </section>
+                    <SearchBar className="collapse navbar-collapse" id="navbarSupportedContent">
+                        <SearchWrapper className="offset-md-3">
+                            <SearchLogo />
+                            <SearchInput type="text" placeholder="Search" onChange={this.props.searchEdit} value={this.props.searchValue}/>
+                        </SearchWrapper>
```
Comment (OmarSalah95) [inline instagram-app/src/components/SearchBar/SearchBar.jsx:30]: Interesting to see you add the logo in the search bar by wrapping it as such. Nice work.

**Review by OmarSalah95 — APPROVED** [review]
Great work Rebuilding Instagram this week.

**Pull request merged**